### PR TITLE
add messaging contract for duck player page

### PR DESCRIPTION
**Pull request by shakyShane**
https://app.asana.com/0/1201141132935289/1207942137400006/f

These are the schema files used by DuckPlayer - split out here for easier review.

### CODE REVIEW

**Comment by shakyShane** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1003?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#993** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/993?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1004** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1004?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1003** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1003?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1002** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1002?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @shakyShane and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/special-pages/messages/duckplayer/openInfo.notify.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (mgurgel) [inline packages/special-pages/messages/duckplayer/openInfo.notify.json:1]: Is ` "$schema": "http://json-schema.org/draft-07/schema#”` needed here?

File: packages/special-pages/messages/duckplayer/openSettings.notify.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (mgurgel) [inline packages/special-pages/messages/duckplayer/openSettings.notify.json:1]: Is ` "$schema": "http://json-schema.org/draft-07/schema#”` needed here?

**Review by mgurgel — APPROVED** [review]
Just two questions, but looks good otherwise!

File: packages/special-pages/messages/duckplayer/openInfo.notify.json (hunk 1 above)
Comment (shakyShane) [reply packages/special-pages/messages/duckplayer/openInfo.notify.json:1]: Ahh yeah, thanks I'll add it in.

**Pull request merged**